crates/sui-graphql-rpc/src/types/stake.rs

```diff
@@ -309,6 +309,11 @@ impl StakedSui {
             .extend()
     }
 
+    /// The id of the validator staking pool this stake belongs to.
```

Comment:
nit: still needs updating.

```suggestion
    /// The Object ID of the validator staking pool this stake belongs to.
```